<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserForm</class>
 <widget class="QWidget" name="UserForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="questionsWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="answersWidget">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>391</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addQuestionBtn">
   <property name="geometry">
    <rect>
     <x>324</x>
     <y>310</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add question</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="questionLE">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>310</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="answerLE">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>310</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addAnswerBtn">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>310</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Answer</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="votesSpin">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>340</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
